BootloaderCommonPkg/Include/Guid/PayloadKeyHashGuid.h

```diff
@@ -14,21 +14,26 @@
 ///
 extern EFI_GUID gPayloadKeyHashGuid;
 
-#ifndef    SHA256_DIGEST_SIZE
-#define    SHA256_DIGEST_SIZE   32
+#ifndef    HASH_DIGEST_MAX
+#define    HASH_DIGEST_MAX   64

```

Comment:
why do we do #ifndef ?   Anybody defined this previously?   If so, we should fix it to use a single definition.

Reply:
This is no longer required. I will remove it.